docs/admin-guide/troubleshooting.md

```diff
@@ -92,9 +92,128 @@ The steps below are applicable for Severity ("**Sev**") 1 & 2 level incidents. S
 
 After receiving a new alert or ticket, spend a few minutes (&lt; 5m) do a preliminary investigation to identify the reproducibility and impact of the incident. Once confirmed as an issue, the formal incident response process is:
 
-### Hypothesis Development
+### Observing the environment
 
-### Hypothesis Testing
```

Comment:
Look for pods with error states
Look at logs
Look at node health

I don't know if we want to talk about where to look being a product of what the error is?  eg a service is unavailable - is the deployment healthy, is the service up and does it have endpoints, is the ingress up, what do deployment logs say, what does ingress backend logs say etc

Reply:
I'm planning on expanding the `### Observing the environment` section to give some help with this.

Reply:
Yeah, there's a couple more first pass things you should be doing in k8s than just checking metrics

Reply:
OK, I'll resolve this once I've added it.